src/o365/teams/commands/report/teams-report-deviceusagedistributionusercounts.ts

```diff
@@ -39,18 +48,62 @@ class TeamsReportDeviceUsageDistributionUserCountsCommand extends GraphCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        cmd.log(res);
+        let content: string = '';
+
+        if (args.options.output && args.options.output.toLowerCase() === 'json') {
+          let reportdata = this.getJsonReport(res);
+          content = JSON.stringify(reportdata);
+
+          if (!args.options.outputFile) {
+            cmd.log(reportdata);
+          }
+        }
+        else {
+          content = res;
+
+          if (!args.options.outputFile) {
+            cmd.log(content);
+          }
+        }
+
+        if (args.options.outputFile) {
+          fs.writeFileSync(args.options.outputFile, content, 'utf8');
```

Comment:
Have you checked if this also works with relative paths without resolving the path first?

Reply:
Yes, It is working with relative path as well.